api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/GroupedPropertyList.java

```diff
@@ -136,12 +140,57 @@ is to make the get methods (getProperty and getPropertyAndRemoveFromList) method
         // Build the template data model from the groupList
         groups = new ArrayList<PropertyGroupTemplateModel>(
                 propertyGroupList.size());
+        
+        
+        if(!parallel) {
```

Comment:
I believe this block will never be executed, because the one caller of this class now only passes in "parallel = true".